lib/nerves_network.ex

```diff
@@ -26,11 +33,19 @@ defmodule Nerves.Network do
           | {:ipv4_subnet_mask, Types.ip_address()}
           | {:domain, String.t()}
           | {:nameservers, [Types.ip_address()]}
-          | {atom, any()}
+          | {:networks, [[wifi_settings]]}
+          | wifi_settings
```

Comment:
I think the formatter wants:
```suggestion
          | wifi_settings()
```